kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/parquet/ParquetSuiteBase.scala

```diff
@@ -0,0 +1,266 @@
+package io.delta.kernel.defaults.internal.parquet
+
+import java.nio.file.{Files, Paths}
+import java.util.Optional
+
+import scala.collection.JavaConverters._
+import scala.util.control.NonFatal
+
+import io.delta.golden.GoldenTableUtils.goldenTableFile
+import io.delta.kernel.Table
+import io.delta.kernel.data.{ColumnarBatch, FilteredColumnarBatch}
+import io.delta.kernel.defaults.utils.{TestRow, TestUtils}
+import io.delta.kernel.expressions.Column
+import io.delta.kernel.internal.util.ColumnMapping
+import io.delta.kernel.internal.util.Utils.toCloseableIterator
+import io.delta.kernel.types.{ArrayType, BinaryType, BooleanType, ByteType, DataType, DateType, DecimalType, DoubleType, FloatType, IntegerType, LongType, MapType, ShortType, StringType, StructType, TimestampType}
+import io.delta.kernel.utils.{DataFileStatus, FileStatus}
+import org.apache.hadoop.fs.Path
+import org.apache.parquet.hadoop.ParquetFileReader
+import org.apache.parquet.hadoop.metadata.ParquetMetadata
+
+trait ParquetSuiteBase extends TestUtils {
+
+  // Read the parquet files in actionFileDir using Spark Parquet reader
+  def readParquetFilesUsingSpark(actualFileDir: String, readSchema: StructType): Seq[TestRow] = {
+    spark.read
+      .format("parquet")
+      .parquet(actualFileDir)
+      .to(readSchema.toSpark)
+      .collect()
+      .map(TestRow(_))
+  }
+
+  def verify(actualFileDir: String, expected: Seq[FilteredColumnarBatch]): Unit = {
```

Comment:
Make sure to copy over the method docs once you rebase this PR on the latest master

Reply:
@vkorukanti I checked again, this method has no previous documentation.

Reply:
The method got renamed: here it is: https://github.com/delta-io/delta/blob/master/kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/parquet/ParquetFileWriterSuite.scala#L324